tensorflow/python/keras/layers/recurrent_v2.py

```diff
@@ -1333,6 +1336,36 @@ def cudnn_lstm(inputs, init_h, init_c, kernel, recurrent_kernel, bias, mask,
   # so that mathematically it is same as the canonical LSTM implementation.
   full_bias = array_ops.concat((array_ops.zeros_like(bias), bias), 0)
 
+  if build_info.is_rocm_build:
```

Comment:
I don't have much knowledge about rocm, what's the relationship between cudnn and it? Is it an extra layer above it?